site/reference/fn/eval/README.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "`eval`"
+linkTitle: "eval"
+type: docs
+description: >
+  Execute function on resources
+---
+
+<!--mdtogo:Short
+    Execute function on resources
+-->
+
+`eval` executes a function on resources in a directory. Functions are packaged
+as container images or binary executables.
```

Comment:
I wouldn't highlight `binary executables` as that's not the main use case. It's mostly for function developers.